ITR/data/base_providers.py

```diff
@@ -285,23 +319,16 @@ def _calculate_target_projections(self, production_bm: BaseProviderProductionBen
             else:
                 base_year_production = next((p.value for p in c.historic_data.productions if
                                              p.year == self.temp_config.CONTROLS_CONFIG.base_year), None)
-                with warnings.catch_warnings():
-                    warnings.simplefilter("ignore")
-                    company_sector_region_info = pd.DataFrame({
-                        self.column_config.COMPANY_ID: [c.company_id],
-                        self.column_config.BASE_YEAR_PRODUCTION: [base_year_production.to(c.production_metric.units)],
-                        self.column_config.GHG_SCOPE12: [c.ghg_s1s2],
-                        self.column_config.GHG_SCOPE3: [c.ghg_s3],
-                        self.column_config.SECTOR: [c.sector],
-                        self.column_config.REGION: [c.region],
-                    }, index=[0])
-                    bm_production_data = (
-                            production_bm.get_company_projected_production(company_sector_region_info)
-                            # We transpose the data so that we get a pd.Series that will accept the pint units as a whole (not element-by-element)
-                            .iloc[0].T
-                            .astype(f'pint[{str(base_year_production.units)}]')
-                            )
-                c.projected_targets = EITargetProjector().project_ei_targets(c, bm_production_data)
+                try:
+                    co_cumprod = df_partial_pp.loc[c.sector, c.region, EScope.S1S2]
```

Comment:
should we change hard-coded `EScope.S1S2` to a `scope` argument now or later?

Reply:
I've already pushed the change to make it AnyScope.  No idea why GitHub is showing "Changes from 44 commits" instead of "All Changes".  I asked it to show me all changes, it re-rendered a bunch of stuff, I changed to "Conversations" to view content that way, and it completely changed the basis of the code.  So...yes it is fixed.